residency.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2017 Ondrej Smola All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package marathon
+
+import "time"
+
+// TaskLostBehaviorType sets action taken when the resident task is lost
+type TaskLostBehaviorType string
+
+const (
+	// TaskLostBehaviorTypeWaitForever sets to don`t take any action when the resident task is lost
+	TaskLostBehaviorTypeWaitForever TaskLostBehaviorType = "WAIT_FOREVER"
+	// TaskLostBehaviorTypeRelaunchAfterTimeout sets to try relaunch the lost resident task
+	// on another node after relaunch escalation timeout
```

Comment:
Rephrase proposal:

`TaskLostBehaviorTypeWaitForever indicates to try relaunching the lost resident task on another node after the relaunch escalation timeout has elapsed`